docs/index-shared.asciidoc

```diff
@@ -0,0 +1,67 @@
+
+:release-state:  unreleased
```

Comment:
Can you preserve the original comment above this to indicate the different possibilities for release-state?

```
//////////
release-state can be: released | prerelease | unreleased
//////////
```